packages/backend/src/tokens/verify.ts

```diff
@@ -36,16 +36,12 @@ export async function verifyToken(token: string, options: VerifyTokenOptions): P
 
   if (jwtKey) {
     key = loadClerkJWKFromLocal(jwtKey);
+  } else if (typeof issuer === 'string') {
+    // Fetch JWKS from Frontend API if an issuer of type string has been provided
+    key = await loadClerkJWKFromRemote({ issuer, kid, jwksCacheTtlInMs, skipJwksCache });
```

Comment:
@chanioxaris If not mistaken, the default issuer we currently pass is a function object, right? 

If the issuer is passed as a function during the verifyToken invocation, is it a function at this point or has the function executed and returned a string already?

Reply:
Yes that's right the issuer can be a string or a function that returns a boolean value. We will only land in this statement if the provided issuer is of type string, so we can construct the well known url. If the issuer instead is a function we will fallback to other cases (Backend API, JWT key)